.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestHelpers.cs

```diff
@@ -248,7 +248,7 @@ public static IDateTimeParser GetEnglishParser(DateTimeParsers parserName)
                 case DateTimeParsers.Holiday:
                     return new BaseHolidayParser(new EnglishHolidayParserConfiguration());
                 case DateTimeParsers.TimeZone:
-                    return new BaseTimeZoneParser();
```

Comment:
No need to rename all of these. "Time zone" is actually two words.
If we do this type of renaming, we also need to be sure the files are renamed appropriately, not only the classes.

Reply:
This rename is also inconsistent with some of the new methods in this PR.